marklogic-data-hub-central/ui/e2e/cypress/integration/scenarios.modeling.tsx

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress"/>
+
+import HomePage from "../support/pages/home";
+import ModelingPage from '../support/pages/modeling';
+
+const homePage = new HomePage();
+const modelingPage = new ModelingPage();
+
+describe('Entity Modeling Page', () => {
+
+  //login with valid account
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains('MarkLogic Data Hub');
+    cy.fixture('users').then(user => {
+      cy.login(user.username, user.password);
+    })
+    cy.wait(500);
+    homePage.getModeling().click();
+  });
+
+  it('can add an entity type', () => {
+    modelingPage.addEntityType();
+    modelingPage.newEntityName('New-Model');
+    modelingPage.newEntityDescription('Description for New-Model');
+    modelingPage.submitNewEntityForm();
+    cy.wait(2000);
+    modelingPage.getEntity('New-Model').should('exist');
+  });
+
+  it('add entity input with invalid name', () => {
```

Comment:
I would like us to avoid writing integration/e2e tests for scenarios that can be covered using RTL. Form validations and table rendering test below can be verified using RTL as unit tests.

Reply:
I agree with this, I removed the integration test scenarios for now.